executor/aggregate_test.go

```diff
@@ -344,11 +352,11 @@ func (s *testSuite1) TestAggregation(c *C) {
 	_, err = tk.Exec("select std_samp(a) from t")
 	// TODO: Fix this error message.
 	c.Assert(errors.Cause(err).Error(), Equals, "[expression:1305]FUNCTION std_samp does not exist")
-	_, err = tk.Exec("select variance(a) from t")
+	//_, err = tk.Exec("select variance(a) from t")
+	//c.Assert(errors.Cause(err).Error(), Equals, "unsupported agg function: variance")
```

Comment:
please do not comment them out, you can simply rewrite the expected behavior of this test.